conv/gtfs_db_to_db.py

```diff
@@ -40,7 +41,7 @@ def gtfs_db_to_db(source_database: Path, target_database: Path, clean_database:
                     # Extract calendar information
                     # gtfs_calendar_generator(db_read, db_write, {})
 
-                    versions = list(db_read.iter_only_objects(txn_read, Version, 1))
+                    versions = list(db_read.iter_only_objects(txn_read, Version, b'1'))
```

Comment:
This is not correct. 1 was not intended as start_key but limit.

```suggestion
                    versions = list(db_read.iter_only_objects(txn_read, Version, limit=1))
```